arangod/V8Server/v8-collection.cpp

```diff
@@ -2140,8 +2130,10 @@ static void JS_TruncateVocbaseCol(v8::FunctionCallbackInfo<v8::Value> const& arg
   v8::HandleScope scope(isolate);
 
   OperationOptions opOptions;
-  opOptions.waitForSync = ExtractBooleanArgument(isolate, args, 1);
-  ExtractStringArgument(isolate, args, 2, opOptions.isSynchronousReplicationFrom);
+
+  opOptions.waitForSync = ExtractBooleanArgument(isolate, args, 1, false);
+  opOptions.isSynchronousReplicationFrom = ""; // not specified anymore.
```

Comment:
This is empty by default anyway, so we can omit assigning the empty string to it.
```suggestion
```